src/functions/formatSort.js

```diff
@@ -14,7 +13,7 @@ module.exports = function formatSort(sortBy, defaults) {
       var sortInstructions = sortInstruction.split(':');
       if (defaults && sortInstructions.length === 1) {
         var similarDefault = find(defaults, function(predicate) {
-          return startsWith(predicate, sortInstruction[0]);
+          return predicate[0] === sortInstruction[0];
```

Comment:
On a single letter? Really? You don't think it's a mistake? 

It would have made more sense to use `sortInstructions` with an `S` rather than `sortInstruction`. WDYT?

Reply:
I was doubting about this too, but forgot when merging, let's rethink this